applications/nrf5340_audio/README.rst

```diff
@@ -1236,15 +1237,24 @@ You can test upgrading the firmware on both cores at the same time on a headset
 
       python buildprog.py -c both -b debug -d headset --pristine -m external
 
-#. Transfer the generated zip file to your Android or iOS device.
-   The file name should start with :file:`dev_headset_build_debug_dfu_application`.
+#. Transfer the generated file to your Android or iOS device.
+      a. For multi-image DFU, the file name follows the pattern ``dev_<Role>_build_<Application version>_dfu_application<Revision>.zip``,
+      For example :file:`dev_headset_build_debug_dfu_application_0.0.0+0.zip`.
+
+      b. For single-image DFU, the file for app core follows the pattern ``dev_<Role>_build_<Application version>_app_update_<Revision>.bin``
+      The file for net core follows the pattern ``dev_<Role>_build_<Application version>_net_core_app_update_<FW_VERSION>.bin``.
+      For example :file:`dev_headset_build_debug_app_update_0.0.0+0.bin`, :file:`dev_headset_build_debug_net_core_app_update_1.bin`.
```

Comment:
The example name for the application core should be moved to step a.

Reply:
Multi-image DFU can use these files, zip, app core binary, net core.
Since we want multi-image to be DFU at once. We should choose zip file.
The app core/net core binary should go to single-image DFU description.

Reply:
I don't understand. Does this mean single-image uses 2 files and multi-image uses 1 file? That is totally counterintuitive :/

Reply:
2 .bin files are being generated for single core DFU (net.bin and app.bin), but only one file .bin file is used at a time